doc/man3/X509_STORE_CTX_new.pod

```diff
@@ -67,22 +67,31 @@ is no longer valid.
 If I<ctx> is NULL nothing is done.
 
 X509_STORE_CTX_init() sets up I<ctx> for a subsequent verification operation.
-It must be called before each call to X509_verify_cert(), i.e. a I<ctx> is only
-good for one call to X509_verify_cert(); if you want to verify a second
+It must be called before each call to L<X509_verify_cert(3)>, i.e., a I<ctx> is
+only good for one call to L<X509_verify_cert(3)>; if you want to verify a second
```

Comment:
`only good for one certificate verification; ....`

Reply:
I thought I had suggested a "dirty" flag in the context.  If you can only verify once, then why doesn't the code check and issue a run-time error for multiple use, just like the NULL checks it does?

Reply:
@richsalz, now I understand what you meant by suggesting a "dirty" flag. 
This is already implemented in `X509_verify_cert()`:
```
    if (ctx->chain != NULL) {
        /*
         * This X509_STORE_CTX has already been used to verify a cert. We
         * cannot do another one.
         */
        ERR_raise(ERR_LIB_X509, ERR_R_SHOULD_NOT_HAVE_BEEN_CALLED);
        ctx->error = X509_V_ERR_INVALID_CALL;
        return -1;
    }
```